src/test/ui/liveness/liveness-move-in-while.stderr

```diff
@@ -1,21 +1,11 @@
-error[E0382]: use of moved value: `y`
-  --> $DIR/liveness-move-in-while.rs:17:24
+error[E0382]: borrow of moved value: `y`
+  --> $DIR/liveness-move-in-while.rs:18:24
    |
 LL |         println!("{}", y); //~ ERROR use of moved value: `y`
-   |                        ^ value used here after move
-LL |         while true { while true { while true { x = y; x.clone(); } } }
-   |                                                    - value moved here
+   |                        ^ value moved here in previous iteration of loop
```

Comment:
Huh, interesting. This doesn't quite look right, since this location is not a move.

Reply:
Is this the current output?

Reply:
I expect to see "value used here after move", and the "value moved here..." annotation somewhere else.